administrator/components/com_admin/sql/updates/mysql/5.3.0-2024-10-13.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `#__fields` MODIFY `fieldparams` MEDIUMTEXT;
```

Comment:
```suggestion
ALTER TABLE `#__fields` MODIFY `fieldparams` MEDIUMTEXT NOT NULL;

```
1. Line feed at the end of file (just a kind of code style).
2. Maybe we should add the `NOT NULL` here in order to play safe, i.e. make sure it is like it should be regardless of possibly different behaviour of different MySQL or MariaDB versions?